styles.css

```diff
@@ -461,3 +464,172 @@ ul.description {
   margin-bottom: 0.938rem;
   border-bottom: 0.063rem solid rgb(186, 183, 183);
 }
+
+.contact-form {
+  border: 1px solid #6070ff;
+  border-radius: 100px 0 0 0;
+  background-color: #6070ff;
+  margin-top: 2rem;
+}
+
+.form-1 {
+  display: flex;
+  flex-direction: column;
+  padding-left: 20px;
+}
+
+.Contact {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-size: 2.5rem;
+  font-weight: 700;
+  font-family: 'Poppins', system-ui, Courier, monospace;
+  font-style: normal;
+  color: #fff;
+  position: absolute;
+  margin-top: 4.375rem;
+  padding-left: 2.813rem;
+}
+
+.application {
+  font-size: 20px;
+  font-weight: 400;
+  font-family: 'Poppins', system-ui, Courier, monospace;
+  font-style: normal;
+  color: #ebebff;
+  text-align: center;
+  margin-bottom: 2.5rem;
+  position: absolute;
+  margin-top: 9.375rem;
+  width: 20.438rem;
+  height: 6.375rem;
+  line-height: 1.875rem;
+}
+
+.name {
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+.form-image {
+  padding-left: 132px;
+  position: absolute;
+}
+
```

Comment:
- [ ] The absolute CSS declaration that you used for this element is preventing me from inputting values in each of the form control. I would advise you to look for a better way to style the form section so that users can input values in each form control. 👍 💻 

![chrome-capture-2023-2-2](https://user-images.githubusercontent.com/77106962/222548409-a93a5238-b396-4b3c-b2a4-10121cef9a64.gif)